src/Stratis.Bitcoin.Features.BlockStore/BlockStoreFeature.cs

```diff
@@ -119,6 +119,11 @@ public Task<uint256> GetTrxBlockIdAsync(uint256 trxid)
             return this.blockRepository.GetTrxBlockIdAsync(trxid);
         }
 
+        public ChainedHeader GetHighestPersistedBlock()
```

Comment:
`<inheritdoc/>`

Reply:
Does this work even though It's a method & not a field of the name HighestPersistedBlock?